### PR TITLE
Only create pull request if more than one file has changed

### PR DESCRIPTION
**Pull request by Nyholm**
Blocked by #272

### CODE REVIEW

File: .github/workflows/watch.yml [hunk 1]
```diff
@@ -42,6 +42,15 @@ jobs:
       run: |
         ./generate --all
 
+    - name: Check if anything changed
+      run: |
+        if [[ $(git diff --numstat | wc -l) -ge 1 ]]; then
+          exit 0;
+        else
+          echo "No significant changes."
+          exit 1;
```
Comment (jderusse) [inline .github/workflows/watch.yml]: `git reset --hard` to let the pipeline finish without error/notification

File: .github/workflows/watch.yml [hunk 2]
```diff
@@ -42,6 +42,15 @@ jobs:
       run: |
         ./generate --all
 
+    - name: Check if anything changed
+      run: |
+        if [[ $(git diff --numstat | wc -l) -ge 1 ]]; then
```
Comment (jderusse) [inline .github/workflows/watch.yml]: `-eq 1` and remove the else part

File: .github/workflows/watch.yml (hunk 1 above)
Comment (Nyholm) [reply .github/workflows/watch.yml]: Smart!

File: .github/workflows/watch.yml (hunk 2 above)
Comment (Nyholm) [reply .github/workflows/watch.yml]: Thank you

**Comment by jderusse** [comment]
LGTM :+1:

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**